### DISCUSSION TITLE
Search options strange behavior

### DISCUSSION THREAD

**Discussion by Muzeau** [Part-DB/Part-DB-server#276] [Q&A]
Thank you for this beautiful work, however If I modify the display of the default columns for example by adding MPN and then I do a search on the latter it does not work, it is as if the search was only carried out on the default layout, as if search options doesn't work.
What I am doing wrong ?

**Comment by jbtronics** [comment]
Strange, that shouldnt happen. Could verify that MPN search doesnt work on demo.
Transfered to issue #277

**Reply by Muzeau** [reply]
Just tested on demo site and found the same behavior. Try searching  1N4004RLG in MPN only tagged.

**Comment by jbtronics** [comment]
Fixed now
